api/peripheral_api/models/RFID.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const RFIDSchema = new Schema(
+  {
+    name: {
+      type: String,
+      required: true,
+    },
+    byte: {
```

Comment:
maybe give a more descriptive thing like `cardData`? or add a comment above this line explaining what `byte` is
```suggestion
    // the actual bytes of a user's RFID card
    byte: {
```